g2p/convert.py

```diff
@@ -16,12 +16,15 @@ class BlissLexiconToG2PLexiconJob(Job):
     Convert a bliss lexicon into a g2p compatible lexicon for training
     """
 
-    def __init__(self, bliss_lexicon):
+    __sis_hash_exclude = {"include_pronunciation_variants": False}
+
+    def __init__(self, bliss_lexicon, include_pronunciation_variants=False):
         """
         :param Path bliss_lexicon:
+        :param bool include_pronunciation_variants: In case of multiple phoneme representations for one lemma, when this is false it outputs only the first phoneme
         """
         self.bliss_lexicon = bliss_lexicon
-
+        self.include_pronunciation_variants = include_pronunciation_variants
```

Comment:
Actually the new line between the input parameters and output variables was OK as these are two different logical blocks. I.e. before you have a newline between `self.bliss_lexicon = ...` and `self.out_g2p_lexicon = ...` because these are two parts of the constructor. As `self.include_pronunciation_variants` is an input member it should be in the first block (without separation of the members within the block), but the second block is still separated by a newline from the first one.